packages/form-core/src/validate/validators/Required.js

```diff
@@ -9,6 +9,25 @@ export class Required extends Validator {
     return 'Required';
   }
 
+  static get _compatibleRoles() {
```

Comment:
Here it would be nice to explain how these relate to: https://www.w3.org/TR/wai-aria/#aria-required